x-pack/platform/plugins/shared/fleet/cypress/reporter_config.json

```diff
@@ -0,0 +1,10 @@
+{
+  "reporterEnabled": "mochawesome, mocha-junit-reporter",
+  "reporterOptions": {
+    "html": false,
+    "json": true,
+    "mochaFile": "../../../../../target/kibana-fleet/cypress/results/TEST-fleet-cypress-[hash].xml",
```

Comment:
Not sure these 2 relative paths are correct.
I imagine the `target` folder should be straight under the root, and I don't think this was the case even before the relocation.
@elastic/fleet Could you please review and fix in a follow-up PR if needed? TIA